client/state/ui/selectors/get-selected-site-id.ts

```diff
@@ -6,6 +7,6 @@ import 'calypso/state/ui/init';
  * @param  {object}  state Global state tree
  * @returns {?number}       Selected site ID
  */
-export default function getSelectedSiteId( state ) {
+export default function getSelectedSiteId( state: IAppState ): number | null {
```

Comment:
Your type here says that this can return null, but the JSDoc says it can return undefined. Do you know which is right? null and undefined are both falsy but they are very different values.

Reply:
Doesn't `?number` type in JSDoc mean that the value can be null or undefined?

At least that's what I'm seeing under the "Nullable Type" Subheading [here in the docs](https://jsdoc.app/tags-type.html)

Reply:
> Doesn't ?number type in JSDoc mean that the value can be null or undefined?

😮 TIL! I assumed it was the same as TypeScript, which does not allow null (but does make it optional).